src/solvers/unsteady_solver.C

```diff
@@ -241,7 +264,7 @@ void UnsteadySolver::retrieve_timestep()
 
 void UnsteadySolver::integrate_adjoint_sensitivity(const QoISet & qois, const ParameterVector & parameter_vector, SensitivityData & sensitivities)
 {
-  // We are using the midpoint rule to integrate each timestep
+  // We are using the trapezoidal rule to integrate each timestep
```

Comment:
Good fix.  In hindsight - is this the right thing to do for all theta?  I'd think that we'd end up with significantly different weightings if we have an adaptive deltat and we want our integration rule to be consistent with the integrator we use for the transient solves.

Reply:
Actually, this function should probably be in Euler/2Solver.C and follow the scheme of integrate_qoi_timestep.

Reply:
Added a comment to this end.